experiment/simulation/js/towerofhanoi.js

```diff
@@ -10,22 +10,22 @@ window.view = {
 	lastRedDiv: {},
```

Comment:
Is this file still relevant since tower of hanoi has been removed ?

Reply:
I didn't remove it, so that in the future, if needed and possible, the other developer can use this file and try to improve the tower of Hanoi code and re-include it in the menu.

Reply:
ok